edx_figures/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
@OmarIthawi Just remembered. I'll rename this file to be more descriptive

Reply:
@johnbaldwin initial is good enough. Subsequent migrations are the tricky ones. Thanks though :)